packages/sdk/src/index.ts

```diff
@@ -32,7 +33,10 @@ export class Unkey {
   }
 
   private async fetch<TResult>(req: ApiRequest): Promise<TResult> {
-    const url = `${this.baseUrl}/${req.path.join("/")}`;
+    let url = `${this.baseUrl}/${req.path.join("/")}`;
+    if (req.params) {
+      url += `${req.params}`;
```

Comment:
do we need to be calling `.join("")` on `req.params` here since its an array?

Reply:
I believe we do. 

FWIW this entire PR has a bunch of other work on it now as we want to make more changes to the SDK